syntax = 'proto3';

message SampleProto {
    string first_name = 1;
    string last_name = 2;
	string email = 3;
}

message Message {
    string message = 1;
}